extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java

```diff
@@ -113,11 +113,20 @@ public static FlywayDataSourceRuntimeConfig defaultConfig() {
     public boolean repairAtStart;
 
     /**
-     * Enable the creation of the history table if it does not exist already.
+     * call baseline when migrate is executed against a non-empty schema with no metadata table.
+     * This schema will then be baselined with the baselineVersion before executing the migrations.
+     * Only migrations above baselineVersion will then be applied
      */
     @ConfigItem
     public boolean baselineOnMigrate;
 
+    /**
+     * true to execute Flyway baseline automatically when the application starts when executed against an empty schema, false
```

Comment:
Apparently this command fails if the `flyway_schema_history` table exists (see https://github.com/quarkusio/quarkus/issues/27734#issuecomment-1238813838).

Reply:
I will add a test with an existing table to be sure

Reply:
@gastaldi , I made some test and here is my conclusion:

if the flyway_schema_history table exist and contains any element, everything is ok.
if the flyway_schema_history does not exist, everything is ok.
if the flyway_schema_history exist but is empty, then flyway.info().current() is null so we do execute the baseline and it is failing. 

I will try to find a way to check if flyway_schema_history already exist in flyway api.